host-src/tool/shim.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * This file is part of the dcload Dreamcast serial loader
+ *
+ * Copyright (C) 2001 Andrew Kieschnick <andrewk@napalm-x.com>
```

Comment:
It's not really correct to be attributing copyright to someone who had nothing to do with this particular file...
```suggestion
```